style.css

```diff
@@ -49,4 +49,74 @@
 
 .header_nav_list a:hover{
     color: rgb(241, 245, 8);
+}
+/* >>>>>>>>>>>.....footer style ..............<<<<<<<<<< */
+footer{
+    display:flex;
+    width: 100%;
+    height: 16%;
+    background: #075B7F;
+    justify-content:space-between;
+    align-items:center;
+    position:absolute;
+    bottom: 0;
+}
+/*>>>>...........footer icons style............<<<<<<*/
+.footerIcons{
+    width:30%;
+    margin-left: 5%;
+}
+.footerIcons ul{
+    list-style: none;
+}
+.footerIcons ul li{
+    display: inline-block;
+    padding: 1% 2%;
+    margin-left: 3%;
+    width: 11%;
+    border-right: 1px solid white;
+}
+.footerIcons ul li a{
+    align-items: center;
+    color:white;
+    padding: 5%;
+} 
+.footerIcons ul li a:hover{
+      color: orangered;
```

Comment:
Use this color instead #F2AF34
color: #F2AF34;

Reply:
done